{"uuid":"ff2bbbbe-b1f1-4d05-b100-f7088f035cdd","name":"Scenario Outline: The user getting error message with invalid python code for Types of Linked List","children":["4219a5ba-6ae2-432c-87e9-3bdf2f58b564"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696883550038,"stop":1696883550038},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696883550181,"stop":1696883550181},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696883550253,"stop":1696883550253},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696883552274,"stop":1696883552276}],"start":1696883549866,"stop":1696883552288}